src/Microsoft.ApiDesignGuidelines.Analyzers/CSharp/CSharpEnumStorageShouldBeInt32.Fixer.cs

```diff
@@ -3,14 +3,21 @@
 using System.Composition;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
 
 namespace Microsoft.ApiDesignGuidelines.Analyzers
 {
-    /// <summary>
+    /// <summary> 
```

Comment:
I dunno what's up with these trailing spaces. You must be using some Super New Editor (TM): "Now with even more white space!" :-)